lib/utils.py

```diff
@@ -329,3 +331,30 @@ def is_balena_app():
     :return: bool
     """
     return bool(getenv('RESIN', False)) or bool(getenv('BALENA', False))
+
+
+def is_wott_integrated():
+    """
+    Chacks if wott-agent installed or not
+    :return:
+    """
+    proc_list = sh.ps('axw')
+    re_wottagent = re.compile(r'^.*wott-agent.*$', re.MULTILINE)
+    result = re.findall(re_wottagent, proc_list.stdout)
+    for match in result:
+        if 'wott-agent daemon' in match:
+            return True
+    return False
+
+
+def get_wott_device_id():
+    """
+    :return: WoTT Device id of this device
+    """
+    metadata_path = os.path.join(WOTT_PATH, 'metadata.json')
+    if os.path.isfile(metadata_path):
+        with open(metadata_path) as metadata_file:
+            metadata = json.load(metadata_file)
+        if 'device_id' in metadata:
+            return metadata['device_id']
+    return '-- can not read device id --'
```

Comment:
This should be logged, not just returned. Also, should be "Could not read WoTT Device ID".

Reply:
fixed